addOns/webuipoc/src/main/pocs/reactWebUI/package.json

```diff
@@ -4,26 +4,29 @@
   "private": true,
   "homepage": ".",
   "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
+    "@mui/icons-material": "^5.15.18",
+    "@mui/material": "^5.15.18",
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "axios": "^1.6.8",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
-    "react-scripts": "^5.0.1"
+    "react-scripts": "^5.0.1",
+    "web-vitals": "^2.1.4"
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "PUBLIC_URL=/reactWebUI/ BUILD_PATH=./dist react-scripts build",
+    "build": "PUBLIC_URL=/web-ui/ BUILD_PATH=./dist react-scripts build",
```

Comment:
The directory needs to match the URL, either revert or rename the directory.

Reply:
The rebasing and conflict resolution of tens of files have messed up some files. @thc202 what should be the best approach to avoid this in future and stay current with main?

Reply:
What's mentioned in https://github.com/zaproxy/zap-extensions/pull/5460#issuecomment-2217438908

Reply:
How often?

Reply:
At the very least once before raising a new PR, after the PR is open it depends on what changes are going on (once a week maybe).